_data/simple-icons.json

```diff
@@ -6566,6 +6566,12 @@
             "source": "http://www.netapp.com/",
             "guidelines": "https://www.netapp.com/company/legal/trademark-guidelines/"
         },
+        {
+            "title": "NetBSD",
+            "hex": "FF6600",
+            "source": "https://www.netbsd.org/gallery/logos.html",
```

Comment:
Can you write the source just with "https://www.netbsd.org" please.

Reply:
Should be done in the latest commit.